src/gutenberg-packages/utils.js

```diff
@@ -9,12 +9,11 @@ export const matcherFromSource = (sourceConfig) => {
 	}
 };
 
-// We have to do this because of the way we are currently bundling the code in
-// this repo, each block gets its own copy of this file, but this won't happen
-// if/when we do this in Gutenberg.
 export const createGlobal = (name, initialValue) => {
-	if (typeof window[name] === 'undefined') {
-		window[name] = initialValue;
-	}
-	return window[name];
+	['wp', 'view', name].reduce((obj, name, i) => {
+		if (typeof obj[name] === 'undefined')
+			obj[name] = i === 2 ? initialValue : {};
```

Comment:
Interesting way to initialize a nested property inside an undefined object. 🙂

I would use the length from the array instead of hardcoding it to check if we have reached the property with the initial value:

```suggestion
	['wp', 'view', name].reduce((obj, name, i, array) => {
		if (typeof obj[name] === 'undefined')
			obj[name] = i === (array.length - 1) ? initialValue : {};
```

Reply:
Yeah, but why calculate `array.length - 1` when it is always 2 anyway. And if someone changes the array length in the future and doesn't update the hardcoded value, this thing won't work at all. So I think it's fine with the hardcoded 2 😄🤷‍♂️

Reply:
I'm going to merge this as it is, but feel free to refute my argument!